app.R

```diff
@@ -358,7 +362,19 @@ server <- function(input, output, session) {
   # Render the race results table
   output$race_results_table <- DT::renderDT({
     
-    datatable(filtered_race_results())
+    datatable(filtered_race_results(),
+              options = list("pageLength" = 20,
+                             "paging" = FALSE,
+                             "scrollY" = '800px',
+                             "autoWidth" = TRUE
+              ),
+              selection = "none"
+    ) |> 
+      formatStyle(
+        'flag', target = 'row', 
+        backgroundColor = styleEqual(c(1), c('green'))
```

Comment:
If it's possible, try to just highlight the column with the fastest laptime instead of the entire row.

Reply:
no idea how to do that @erictsai1208

Reply:
No problem. This can be a future enhancement.